docs/devices/8719514440937_8719514440999.md

```diff
@@ -67,3 +67,21 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 The minimal value is `0` and the maximum value is `255`.
 The unit of this value is `lqi`.
 
+### Time (numeric) (dial only)
+Raw value that represents the amount the dial was turned.
+Value can be found in the pusbished state on the `time` property.
+It's not possible to read (`/get`) or write (`/set`) this value.
+The minimal value is `0` and the maximum value is `255`.
+One step amount to `15`.
+
+### Direction (enum) (dial only)
+Direction in which the dial was turned.
+Value can be found in the published state on the `direction` property.
+It's not possible to read (`/get`) or write (`/set`) this value.
+Possible values are: `left` and `right`.
+
+### Type (enum) (dial only)
+Type of the rotation. First message reports as `step` and consuquest messages reports as `rotate` value.
+Value can be found in the published state on the `type` property.
+It's not possible to read (`/get`) or write (`/set`) this value.
```

Comment:
This part of the doc is automatically generated based on the [`exposes`](https://github.com/Koenkk/zigbee-herdsman-converters/blob/ca799050c761ff8268e57227db36e6beffe8d35c/src/devices/philips.ts#L2873) in the zigbee-herdsman-converters device definition, add them there instead.